sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/CreateNewTaskSteps.java

```diff
@@ -57,6 +57,23 @@ public CreateNewTaskSteps(WebDriverHelpers webDriverHelpers, TaskService taskSer
           webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
         });
 
+    When(
+        "^I create a new task with specific data for an event$",
+        () -> {
+          task = taskService.buildGeneratedTaskForEvent();
+          selectTaskType(task.getTaskType());
+          fillSuggestedStartDate(task.getSuggestedStartDate());
+          fillSuggestedStartTime(task.getSuggestedStartTime());
+          fillDueDateDate(task.getDueDateDate());
+          fillDueDateTime(task.getDueDateTime());
+          selectAssignedTo(task.getAssignedTo());
+          selectPriority(task.getPriority());
+          fillCommentsOnTask(task.getCommentsOnTask());
+          fillCommentsOnExecution(task.getCommentsOnExecution());
+          selectTaskStatus(task.getTaskStatus());
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
```

Comment:
This is duplicate code, and we can create a separate method to reduce the code duplication, we can provide as parameter the task.